src/main/java/org/opensearch/performanceanalyzer/reader/ReaderMetricsProcessor.java

```diff
@@ -516,26 +519,27 @@ private void emitShardRequestMetrics(
         }
     }
 
-    private void emitMasterMetrics(long prevWindowStartTime, MetricsDB metricsDB) {
+    private void emitClusterManagerMetrics(long prevWindowStartTime, MetricsDB metricsDB) {
 
-        if (masterEventMetricsMap.containsKey(prevWindowStartTime)) {
+        if (clusterManagerEventMetricsMap.containsKey(prevWindowStartTime)) {
 
-            MasterEventMetricsSnapshot preMasterEventSnapshot =
-                    masterEventMetricsMap.get(prevWindowStartTime);
-            MetricsEmitter.emitMasterEventMetrics(metricsDB, preMasterEventSnapshot);
+            ClusterManagerEventMetricsSnapshot preClusterManagerEventSnapshot =
+                    clusterManagerEventMetricsMap.get(prevWindowStartTime);
+            MetricsEmitter.emitClusterManagerEventMetrics(
+                    metricsDB, preClusterManagerEventSnapshot);
```

Comment:
nit : can be a single line
```suggestion
            MetricsEmitter.emitClusterManagerEventMetrics(metricsDB, preClusterManagerEventSnapshot);
```